brokenspoke_analyzer/core/compute.py

```diff
@@ -33,7 +33,7 @@ def execute_sqlfile_with_substitutions(
         binding_names = sorted(bind_params.keys(), key=len, reverse=True)
         for binding_name in binding_names:
             param = bind_params[binding_name]
-            substitute = param if param else "NULL"
+            substitute = param if param != None else "NULL"
```

Comment:
When comparing to `None`, always use `is` or `is not`. See https://stackoverflow.com/questions/2209755/python-operation-vs-is-not for reference.
```suggestion
            substitute = param if param is not None else "NULL"
```